src/api/app/views/webui2/webui/user/_info.html.haml

```diff
@@ -2,11 +2,14 @@
   .card-body
     = webui2_user_image_tag(user, 'home-avatar')
 
-    %h4.home-usertitle.mt-4.mb-3
+    %h4.home-usertitle.mt-4
       #home-realname
         = user.realname
-      .mt-2#home-login
-        = user.login
+    %h4.d-inline-block#home-login
+      = user.login
+    - role_titles.each do |title|
+      %span.badge.badge-secondary.align-top
```

Comment:
I would add `mt-1` here to align this with the user login

Reply:
Actually I like it a lot more when it is aligned to the top. It is additional information to the user login. 

![selection_008](https://user-images.githubusercontent.com/3799140/49792689-a5b40d00-fd33-11e8-9bcc-1d9d98c1858b.png)

vs

![selection_007](https://user-images.githubusercontent.com/3799140/49792699-a9e02a80-fd33-11e8-9915-9275669af87d.png)